lib/ditcher.js

```diff
@@ -108,9 +110,11 @@ function buildQuery(query, fields, expression) {
         var queryField = {};
         if ('undefined' !== typeof query[field]) {
           queryField = query[field];
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "queryField" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1854)